src/test/java/com/thebois/models/beings/BeingTests.java

```diff
@@ -123,7 +135,12 @@ public void getPathReturnsEqualPositionsToPathFinder() {
         final Position endPosition = new Position(123, 456);
         final List<Position> actualPath = List.of(endPosition);
         when(pathFinder.path(any(), any())).thenReturn(actualPath);
-        final AbstractBeing being = new Pawn(new Position(), endPosition, mockRandom, pathFinder);
+        final AbstractBeing being = new Pawn(
+            new Position(),
+            endPosition,
+            mockRandom,
+            pathFinder,
+            new World(10, 0));
```

Comment:
If the test doesn't need a real world, you probably shouldn't use a real world. Instead, create a mock of the interface that is being passed as an argument, e.g. `mock(IFinder.class)`. This also gives the benefit of tests not having to change when we update the constructor of world.